### DISCUSSION TITLE
Multiple Documents in GitHub Student Developer pack

### DISCUSSION THREAD

**Discussion by PraneetDixit** [community/community#22666] [New to GitHub]
<p>I recently applied for github student developer plan and I was asked to provide my id or some documentation that prove that I am the student of mentioned school. I uploaded a document.<br>
But I also wanted to add a certificate issued to me in which my school was mentioned. But the maximum upload allowed was only 1. I submitted my request at that time but I think that adding the certificate could increase the chance of my application to be accepted.<br>
What can I do so that I can also mention my certificate? (My application is currently under process).<br>
Thanks in advanced.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Sorry for the delay here. It looks as though this was lost in another category!</p>
<p>You can always send in additional documentation if it’s required by opening a ticket with support:</p>
<p><a href="https://support.github.com/contact" class="onebox" target="_blank">https://support.github.com/contact</a></p>
<p>Please be aware this is the busiest time of the year for our Education team, and they will process all applications in order.</p>

**Comment by khatrijay28** [comment]
Hey, Recently I submitted 2 requests for the GitHub pack simultaneously. One gets approved and the second is rejected what does that mean? Can I send a request again....?
